packages/big-design-patterns/src/components/ActionBar/ActionBar.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, ButtonProps, excludeMarginProps } from '@bigcommerce/big-design';
+import React from 'react';
+
+import { warning } from '../../utils';
+
+import { StyledActionBar, StyledActionBarContents } from './styled';
+
+interface ActionButtonProps extends Omit<ButtonProps, 'children' | 'mobileWidth'> {
+  text: string;
+}
+
+interface ActionProps {
+  actions: ActionButtonProps[];
+}
+
+export interface ActionBarProps {
+  actions: ActionButtonProps[];
+}
+
+function validateActions(actions: ActionButtonProps[]) {
+  if (actions.length > 3) {
+    warning('Action bar should not have more than 3 actions.');
+  }
+
+  const primaryButtonActions = actions.filter((action) => {
+    return action.variant === 'primary';
+  });
+
+  if (primaryButtonActions.length > 1) {
+    warning('Action bar should not have more than 1 primary action.');
+  }
+}
+
+const Actions = ({ actions }: ActionProps) => {
```

Comment:
So we don't need to define another interface
```suggestion
const Actions = ({ actions }: Pick<ActionBarProps, 'actions'>) => {
```